packages/functional-tests/lib/ci-reporter.ts

```diff
@@ -13,6 +13,29 @@ import {
   TestResult,
 } from '@playwright/test/reporter';
 
+/**
+ * Converts milliseconds to human-readable format
+ * If the time is less than 1 second, it shows milliseconds
+ * If the time is less than 1 minute, it shows seconds
+ * And if over 1 minute, it shows minutes and seconds
+ * @param ms
+ */
+const formatTime = (ms: number) => {
+  // protect against bad input so we don't crash the reporter
+  if (ms === 0 || ms === undefined || ms === null || isNaN(ms)) {
```

Comment:
This is probably good enough as it is, but noting that this doesn't cover negative numbers, and perhaps ms === 0 could be reported as `0ms`?